README.md

```diff
@@ -101,7 +100,7 @@ See [the elections documentation](elections) for further details.
 
 ### Architecture Committee Meetings
 
-The Architecture Committee meets every Tuesdays at 7:00am PST. Agenda & call info can be found [here](https://etherpad.opendev.org/p/Kata_Containers_2021_Architecture_Committee_Mtgs)
+The Architecture Committee meets every Tuesdays at 7:00am PST. Agenda & call info can be found [here](https://etherpad.opendev.org/p/Kata_Containers_2022_Architecture_Committee_Mtgs)
```

Comment:
Please, point to this address instead: https://etherpad.opendev.org/p/Kata_Containers_Architecture_Committee_Mtgs

Reply:
@zmlcc Can you add a dot(`.`) at the end of this line by the way.

Reply:
> Please, point to this address instead: https://etherpad.opendev.org/p/Kata_Containers_Architecture_Committee_Mtgs

Thanks for reminding. Have changed to the recommended address

Reply:
> @zmlcc Can you add a dot(`.`) at the end of this line by the way.

fixed